src/modules/teachers/components/AssignmentForm.jsx

```diff
@@ -63,21 +69,23 @@ class AssignmentForm extends Component {
     //WildCam Gorongosa has no Staging data that we can use to create Subjects.
     //Therefore, we need to hardcode some Subject IDs.
     //--------
-    /*if (window.location.hostname === 'localhost') {
+    if (window.location.hostname === 'localhost') {
       newAssignment.subjects = [
           '4077',
           '4079',
           '4078',
           '4076',
           '4080',
-          '4075'
+          '4075',
         ];
-    }*/
+    }
     //--------
     
     //NOTE: According to Marten, it's perfectly OK to create a subject with no students or subjects.
     //--------
     if (newAssignment.students.length > 0 && newAssignment.subjects.length) {
+      sessionStorage.setItem('savedNewAssignment', null);
+      sessionStorage.setItem('savedClassroomId', null);
```

Comment:
I derped here too; never use `sessionStorage.setItem('whatever', null);` because `sessionStorage.getItem('whatever')` will return `"null"`, the string. Same with `undefiend`.

This should be `sessionStorage.removeItem('whatever')`.